exercises/test/config/karma.common.js

```diff
@@ -0,0 +1,78 @@
+import _ from 'underscore';
+import path from 'path';
+
+export default {
+  basePath: '../../',
+  frameworks: ['mocha', 'chai', 'chai-sinon', 'phantomjs-shim'],
+  browsers: ['PhantomJS'],
+  reporters: ['mocha'],
+
+  files: [
+    'test/all-tests.coffee',
```

Comment:
lots of places looking for .coffee or .cjsx extensions are probably all broken now